site/README.md

```diff
@@ -7,7 +7,7 @@ the site is to use docker to use a standard environment.
 ## Setup
 
 1. `cd site`
-2. `git clone git@github.com:apache/orc.git -b asf-site target`
```

Comment:
Since this is for developers, I'd think that the ssh version is better.

Reply:
Sure, updated